js/gfx-animation.js

```diff
@@ -0,0 +1,20 @@
+const header = document.getElementById('main-top-header');
+const headGfx = document.querySelectorAll('.header-gfx');
+
+const setAttrSpeed = () => {
+  const speed = Math.floor(Math.random() * (-10 - 10) + 10);
+  return speed !== 0 ? speed : setAttrSpeed();
+};
+
+headGfx.forEach((gfx) => {
+  gfx.setAttribute('data-speed', setAttrSpeed());
+});
+
+header.addEventListener('mousemove', (e) => {
+  headGfx.forEach((gfx) => {
+    // const speed = Math.floor(Math.random() * (-10 - 10) + 10);
```

Comment:
- [ ] please try to remove the unused commented code to keep your file clean and professional